numpy/core/tests/test_arrayprint.py

```diff
@@ -34,6 +34,27 @@ class sub(np.ndarray): pass
             "     [(1,), (1,)]], dtype=[('a', '<i4')])"
         )
 
+    @dec.knownfailureif(True, "See gh-10544")
```

Comment:
So this test is supposed to fail? I'm updating this for pytest to `xfail`, does that make sense. Wouldn't it make more sense to write a test that succeeds?

Reply:
It is expected to fail, I think `xfail` makes sense.

We had previously fixed it so it worked,, but then had to revert those changes because the fix caused a different back-compat bug. I left the test in because it would be nice to one day have it work, in which case we can just re-enable to thest.